src/state/account/accountService.ts

```diff
@@ -16,25 +16,31 @@
  */
 import _ from 'lodash/fp';
 
-import * as terrasoApi from 'terrasoBackend/api';
+import * as terrasoApi from 'state/terrasoBackend/api';
 import {
+  UserFragment,
```

Comment:
i defined a `User` type in `accountSlice.ts`, the `UserFragment` corresponds to what is in our GraphQL API, and the `User` type is the representation in the redux store. In general I think it's good to decouple these, and right now they are in fact different types (`UserFragment` has edges and nodes, `User` does not since they are flattened). `UserFragment` is a type that could be auto-generated if we decide to set up code generation on our GraphQL schema and queries

not sure if it will help with the confusion, but the standard naming convention in Typescript is that types are capitalized and values are lower-cased. So `xFragment` is a graphql fragment, and `XFragment` is the Typescript type of the corresponding object that would be created by including that fragment in a query. is there a different naming convention that would make more sense?

Reply:
That makes sense if potentially, we can even create the GraphQL fragments from those types then this makes sense

Reply:
it would go the other way (creating types from GraphQL fragments) but yeah we could avoid the code dup